### PR TITLE
feat(dashboards): Added Breadcrumbs to the Dashboard Details page

### PR DESCRIPTION
**Pull request by shruthilayaj**
Dashboard details page is now a sub page for business plan
and has breadcrumbs. The manage page will be the main container
for business plan. for Team plans however, the default dashboard
is still the main page. User now also won't be able to delete the last
dashboard.

### CODE REVIEW

File: static/app/views/dashboardsV2/controls.tsx [hunk 1]
```diff
@@ -67,6 +67,7 @@ class Controls extends React.Component<Props> {
             priority="danger"
             message={t('Are you sure you want to delete this dashboard?')}
             onConfirm={onDelete}
+            disabled={!(dashboards.length > 1)}
```
Comment (dashed) [inline static/app/views/dashboardsV2/controls.tsx:70]: @shruthilayaj We also may need to prevent deleting the last dashboard for the API endpoint.

**Review by doralchan — APPROVED** [review]
Looks good from the design end

File: static/app/views/dashboardsV2/controls.tsx (hunk 1 above)
Comment (Zylphrex) [reply static/app/views/dashboardsV2/controls.tsx:70]: Nit: Can we simplify the negation here and just do dashboards.length <= 1?

Comment (shruthilayaj) [reply static/app/views/dashboardsV2/controls.tsx:70]: lol sure thing

**Review by dashed — APPROVED** [review]

**Pull request merged**